web/components/templates/requests/requestsPage.tsx

```diff
@@ -81,12 +89,41 @@ const RequestsPage = (props: RequestsPageProps) => {
       request_user_id: d.request_user_id,
       model: d.response_body?.model,
       temperature: d.request_body?.temperature,
+      ...updated_request_properties,
     };
   });
 
   const hasPrevious = page > 1;
   const hasNext = to <= count!;
 
+  const makeColumn = (name: string) => {
+    return (
+      <th
+        scope="col"
+        className="whitespace-nowrap px-2 py-3.5 text-left text-sm font-semibold text-gray-900"
+      >
+        {name}
+      </th>
+    );
+  };
+
+  const makeValue = (val: string) => {
+    return (
+      <td className="whitespace-nowrap px-2 py-2 text-sm text-gray-500">
+        {val}
+      </td>
+    );
+  };
+
+  const makeCardProperty = (name: string, val: string) => {
+    return (
+      <li className="w-full flex flex-row justify-between gap-4 text-sm">
+        <p>{name}:</p>
+        <p>{val == null ? "{NULL}" : val}</p>
```

Comment:
nit: triple equals. `val === null`. 

also the typical equivalent syntax would be: 
`val || "{null}"`

Reply:
done